app/controllers/auth0_controller.rb

```diff
@@ -41,6 +42,20 @@ def failure
     logout
   end
 
+  # Handle omniauth errors coming from Auth0.
+  def omniauth_failure
+    error = params["error"]
+
+    # Display 'unauthorized' errors but go to `failure` endpoint for all others.
+    if error.present? && error == AUTH0_UNAUTHORIZED
+      description = params["error_description"]
```

Comment:
I believe we need sanitize this input and remove any HTML tags from it to avoid script injections.

Reply:
added a sanitize 👍